integration-tests/src/test/java/oracle/weblogic/kubernetes/ItDBOperator.java

```diff
@@ -0,0 +1,740 @@
+        = assertDoesNotThrow(() -> getCurrentIntrospectVersion(fmwDomainUid, fmwDomainNamespace));
+    logger.info("Before restarting operator introspector pod name is: {0}, introspectVersion is: {1}",
+        introspectPodName, introspectVersion1);
+
+    logger.info("Restarting operator in the namespace: " + opNamespace);
+    restartOperator(opNamespace);
+    //verify the exact same introspector pod exists and Version does not change
+    logger.info("Checking the exact same introspector pod {0} exists in the namespace {1}",
+        introspectPodName, fmwDomainNamespace);
+    checkPodExists(introspectPodName, fmwDomainUid, fmwDomainNamespace);
+    String introspectVersion2
+        = assertDoesNotThrow(() -> getCurrentIntrospectVersion(fmwDomainUid, fmwDomainNamespace));
+    logger.info("After operator restart introspectVersion is: " + introspectVersion2);
+    assertEquals(introspectVersion1, introspectVersion2, "introspectVersion changes after operator restart");
+
+    verifyDomainReady(fmwDomainNamespace, fmwDomainUid, replicaCount);
+    // Expose the admin service external node port as  a route for OKD
+    adminSvcExtHost = createRouteForOKD(getExternalServicePodName(fmwAdminServerPodName), fmwDomainNamespace);
+    verifyEMconsoleAccess(fmwDomainNamespace, fmwDomainUid, adminSvcExtHost);
+
+    //"Reuse the same RCU schema to restart JRF domain"
+    testReuseRCUschemaToRestartDomain();
+
+  }
+
+  /**
+   * Save the OPSS key wallet from a running JRF domain's introspector configmap to a file Restore the OPSS key wallet
+   * file to a Kubernetes secret. Shutdown the domain. Using the same RCU schema to restart the same JRF domain with
+   * restored OPSS key wallet file secret. Verify Pod is ready and service exists for both admin server and managed
+   * servers. Verify EM console is accessible.
+   */
+  private void testReuseRCUschemaToRestartDomain() {
+    saveAndRestoreOpssWalletfileSecret(fmwDomainNamespace, fmwDomainUid, opsswalletfileSecretName);
+    shutdownDomain();
+    patchDomainWithWalletFileSecret(opsswalletfileSecretName);
+    startupDomain();
+    verifyDomainReady(fmwDomainNamespace, fmwDomainUid, replicaCount);
+  }
+
+  /**
+   * Create WebLogic domain using model in image and Oracle database used for JMS and JTA migration and service logs.
+   */
+  @Test
+  void  testWlsModelInImageWithDbOperator() {
+
+    // create secret for admin credentials
+    logger.info("Create secret for admin credentials");
+    String adminSecretName = "weblogic-credentials";
+    assertDoesNotThrow(() -> createDomainSecret(adminSecretName,
+        ADMIN_USERNAME_DEFAULT, ADMIN_PASSWORD_DEFAULT, wlsDomainNamespace),
+        String.format("createSecret failed for %s", adminSecretName));
+
+    // create encryption secret
+    logger.info("Create encryption secret");
+    String encryptionSecretName = "encryptionsecret";
+    assertDoesNotThrow(() -> createDomainSecret(encryptionSecretName, "weblogicenc",
```

Comment:
use constants instead of hardcode the username and password here.

Reply:
We do not have constants for encryption secret/password. We have only for admin user/password.

Reply:
done